client/pdo/client/scripts/eservicedatabaseCLI.py

```diff
@@ -0,0 +1,282 @@
+        # removes all entries corresponding to names and urls
+        
+        num_removed = 0 
+
+        if config['Service'].get('EnclaveServiceNames'):
+            names = config['Service']['EnclaveServiceNames']
+            logger.info(names)
+            for name in names:
+                num_removed += db.remove_info(name = name)
+        
+        if config['Service'].get('EnclaveServiceURLs'):
+            urls = config['Service']['EnclaveServiceURLs']
+            logger.info(urls)
+            for url in urls:
+                num_removed +=  db.remove_info(url = url)
+
+        # save as json file
+        try:
+            db.save_data_to_file()
+        except Exception as e:
+            logger.error('Unable to remove entries: Failed to save as json file' + str(e))
+            sys.exit(-1)
+
+        logger.info('Removed ' + str(num_removed) + ' entries from the database')
+
+
+
+## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+## DO NOT MODIFY BELOW THIS LINE
+## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+ContractHost = os.environ.get("HOSTNAME", "localhost")
+ContractHome = os.environ.get("PDO_HOME") or os.path.realpath("/opt/pdo")
+ContractEtc = os.path.join(ContractHome, "etc")
+ContractKeys = os.path.join(ContractHome, "keys")
+ContractLogs = os.path.join(ContractHome, "logs")
+ContractData = os.path.join(ContractHome, "data")
+LedgerURL = os.environ.get("PDO_LEDGER_URL", "http://127.0.0.1:8008/")
+ScriptBase = os.path.splitext(os.path.basename(sys.argv[0]))[0]
+
+config_map = {
+    'base' : ScriptBase,
+    'data' : ContractData,
+    'etc'  : ContractEtc,
+    'home' : ContractHome,
+    'host' : ContractHost,
+    'keys' : ContractKeys,
+    'logs' : ContractLogs,
+    'ledger' : LedgerURL
+}
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+def Main(commands) :
+    # parse out the configuration file first
+    conffiles = [ 'pcontract.toml' ]
+    confpaths = [ ".", "./etc", ContractEtc ]
+
+    parser = argparse.ArgumentParser()
+    
+    parser.add_argument('--config', help='configuration file', nargs = '+')
+    parser.add_argument('--config-dir', help='configuration file', nargs = '+')
+
+    parser.add_argument('--eservice-url', help='service urls',  nargs='+')
+    parser.add_argument('--eservice-name', help='service names',  nargs='+')
+    parser.add_argument('--eservice-db', help='json file for database', type=str)
+    parser.add_argument('--loglevel', help='Set the logging level', default='INFO')
+    parser.add_argument('--logfile', help='Name of the log file', default='__screen__')
+
```

Comment:
given that you are breaking this into commands, there are much better ways to set up argument processing. See the way we handle arguments in the pdo-shell.